src/main/java/meteordevelopment/meteorclient/systems/hud/elements/InventoryHud.java

```diff
@@ -35,21 +39,76 @@ public class InventoryHud extends HudElement {
         .build()
     );
 
+    interface SupportedSourcesInterface { ArrayList<ItemStack> getItems(); }
```

Comment:
Interface is unnecessary, define the method in the enum instead. (Or modify the system to allow adding sources from third party addons, but that's more tricky)

Reply:
Added `public abstract ArrayList<ItemStack> getItems();` method. Removed an interface.